vowpalwabbit/memory_tree.cc

```diff
@@ -1299,15 +1300,14 @@ base_learner* memory_tree_setup(options_i& options, vw& all)
         tree, as_singleline(setup_base(options, all)), learn, predict, num_learners, prediction_type_t::multilabels);
 
     // all.p->lp = MULTILABEL::multilabel;
```

Comment:
[Minor] Should the commented-out code be removed?

Reply:
Most likely! I'll go ahead and get rid of it